test/configCases/url/path/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	mode: "development",
+	output: {
+		moduleFilename: "images/file.[ext]"
```

Comment:
You should also be able to use `[basename]` and `[file]` too

Reply:
`[basename]` or `[name]` ?  `[name]` doesn't currently work as it is 'reserved'  for chunks and not available in this context, the question is if it would be possible to support common placeholders like `[id]`, `[name]`, `[hash]`, `[ext]` for each context and scope them `if (isModuleFilename) '[hash]' === '[modulehash]'` and so on (but this can be considered later, I will the address the other comments first). I tried `[file]` but it didn't work (I will look into it) :)